api_tests/src/ledgers/bitcoind_instance.ts

```diff
@@ -165,6 +165,7 @@ zmqpubrawtx=tcp://127.0.0.1:${this.zmqPubRawTxPort}
 [regtest]
 bind=0.0.0.0:${this.p2pPort}
 rpcbind=0.0.0.0:${this.rpcPort}
+wallet=miner
```

Comment:
Why are we setting wallet related functionality here? I would prefer to keep this part of the code unaware of wallets. Can we create a `miner` wallet whereever we need it and make sure there is a dataflow one can trace?

Like:

- Initialize the bitcoin miner with a specific wallet name
- Use the same variable to initialize the `BitcoindWallet` to make sure it can spend the money

Reply:
See response on the PR.